contributors/devel/README.md

```diff
@@ -49,10 +49,10 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## Developing against the Kubernetes API
 
-* The [REST API documentation](../api-reference/README.md) explains the REST
+* The [REST API documentation](http://kubernetes.io/docs/reference/) explains the REST
```

Comment:
Lets fix the broken link first.  Since we have several api documentation pages, it isn't clear where it should link to.